openlibrary/catalog/get_ia.py

```diff
@@ -46,7 +47,7 @@ def bad_ia_xml(identifier):
     # need to handle 404s:
     # http://www.archive.org/details/index1858mary
     loc = "{0}/{0}_marc.xml".format(identifier)
-    return '<!--' in urlopen_keep_trying(IA_DOWNLOAD_URL + loc).read()
+    return '<!--' in urlopen_keep_trying(IA_DOWNLOAD_URL + loc).content
```

Comment:
I'm not sure how to trigger this deprecated code, in fact I'm not sure this function does anything correct: comments in a MARC XML are not necessarily mean the data is bad.

but  as it is written, it's looking for a `str` in `bytes` which will error.

Reply:
A grep shows this function is not called in openlibrary, so it can probably be deleted.   I changed to use `.text` for the string comparison.